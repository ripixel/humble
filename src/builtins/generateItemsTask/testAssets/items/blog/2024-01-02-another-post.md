This is another test post for generateItemsTask.

With a second paragraph.
