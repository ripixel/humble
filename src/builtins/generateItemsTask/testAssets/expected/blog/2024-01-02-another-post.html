<html>
  <body>
    <header>
      <h1>Title from header: Another Post</h1>
    </header>
    <main>
      <h2>Title from template: Another Post</h2>
      <article>
        <p>This is another test post for generateItemsTask.</p>
        <p>With a second paragraph.</p>
      </article>
      <footer>Published: 2 January 2024</footer>
    </main>
    <footer>Copyright Test</footer>
  </body>
</html>
